### PR TITLE
Update stripe card object

### PR DESCRIPTION
**Pull request by andreevlex**
part issue #3

### CODE REVIEW

File: stripe/src/resources/card.rs [hunk 1]
```diff
@@ -37,25 +29,89 @@ impl<'a> ::serde::Serialize for CardParams<'a> {
...
+    #[serde(rename = "debit")]
+    Debit,
+    #[serde(rename = "prepaid")]
+    Prepaid,
+    #[serde(rename = "unknown")]
+    Unknown,
+}
+
+#[derive(Deserialize, Serialize, PartialEq, Debug)]
+pub enum TokenizationMethod {
```
Comment (kestred) [inline stripe/src/resources/card.rs]: Maybe add an `Other` variant here, in the likely case that other NFC payment methods are added?

File: stripe/src/resources/card.rs [hunk 2]
```diff
@@ -37,25 +29,89 @@ impl<'a> ::serde::Serialize for CardParams<'a> {
...
+    pub cvc_check: Option<CheckResult>,
+    pub default_for_currency: Option<bool>,
+    pub dynamic_last4: Option<String>,
     pub exp_month: u32,
     pub exp_year: u32,
     pub fingerprint: String,
-    pub funding: String, // (credit, debit, prepaid, unknown)
+    pub funding: CardType,
     pub last4: String,
+    pub metadata: HashMap<String, String>,
```
Comment (kestred) [inline stripe/src/resources/card.rs]: Use `Metadata` from the `params` mod instead (a type alias)

File: stripe/src/resources/card.rs [hunk 3]
```diff
@@ -1,30 +1,22 @@
+use std::collections::HashMap;
+
+use super::currency::Currency;
```
Comment (kestred) [inline stripe/src/resources/card.rs]: Other files follow the pattern `use resources::Currency;`, so lets do that for consistency.

(Until we decide to upgrade to rust2018 😆 )

File: stripe/src/resources/card.rs [hunk 4]
```diff
@@ -37,25 +29,89 @@ impl<'a> ::serde::Serialize for CardParams<'a> {
...
+    #[serde(rename = "fail")]
+    Failed,
+    #[serde(rename = "unavailable")]
+    Unavailable,
+    #[serde(rename = "unchecked")]
+    Unchecked,
+}
+
+#[derive(Deserialize, Serialize, PartialEq, Debug)]
+pub enum CardBrand {
```
[the remark this answers is not in the document]
Comment (andreevlex) [reply stripe/src/resources/card.rs]: I am used stripe api docs for create variants this enum

File: stripe/src/resources/card.rs (hunk 1 above)
Comment (andreevlex) [reply stripe/src/resources/card.rs]: I am used stripe api docs for create variants this enum

**Pull request merged**